service/collector/helper.go

```diff
@@ -62,7 +61,7 @@ func newHelper(config helperConfig) (*helper, error) {
 func (h *helper) GetARNs() ([]string, error) {
 	var arns []string
 
-	clusterCRList, err := h.cmaClient.ClusterV1alpha1().Clusters(corev1.NamespaceAll).List(v1.ListOptions{})
+	clusterCRList, err := h.g8sClient.InfrastructureV1alpha2().AWSClusters().List(metav1.ListOptions{})
```

Comment:
There was now this change in generated clients that CR namespace is not defined anymore. We need to have a close eye to ensure this doesn't introduce any unexpected side-effects.